libr/util/r_rand.c

```diff
@@ -0,0 +1,20 @@
+/* radare2 - LGPL - Copyright 2018 */
+
+#include <r_util.h>
+
+R_API void r_srand (int seed) {
```

Comment:
there's no need to expose that api, we can use defines. or move it into libr/util/unum.c and name it `r_num_srand, rand,..`